src/components/ConnectWallet/ConnectWalletModal.tsx

```diff
@@ -146,12 +171,12 @@ export const DisconnectWalletModal: FC<DisconnectWalletModalProps> = ({
           <Address>{address}</Address>
         </div>
         <div className="flex items-center justify-between py-5">
-          <div className={clsxm({ "text-black": isCopied === true })}>
-            <BiCopy
-              onClick={() => staticCopy(address as string)}
-              className="mr-2 inline h-5 w-5"
-            />
-            <span>Copy address</span>
+          <div
+            className={clsxm({ "text-green-400": isCopied === true })}
```

Comment:
I feel like the green looks a little out of place here, but I'm not sure what would be better. Maybe @galangster has a suggestion?